### ISSUE TITLE
Android `webgl` backend outputs are incorrect and inconsistent between devices

### ISSUE THREAD

**Issue by gnsmrky** [Microsoft/onnxjs#116]
On Google Pixel 3, `Pad 'Reflection'` with `webgl` backend outputs zeros.
On Huawei Mate 20 Pro, `Pad 'Reflection'` is correct, but `Conv2D` output is not.

Is there a requirement for `webgl` on Android devices?  

I've created a quick test page at [ONNX Op test page (https://gnsmrky.github.io/pytorch-fast-neural-style-onnxjs/test_ops.html)](https://gnsmrky.github.io/pytorch-fast-neural-style-onnxjs/test_ops.html).  Can someone have a quick look at the above issues?  I am trying to find a combination that works across different Android devices using `webgl` backend.

**Comment by fs-eire** [comment]
@gnsmrky I tried to reproduce this issue using Android emulator on PC. However the issue is not shown in the emulator. I think I need a real device to test and debug (I will get one in couple of days).

**Comment by gnsmrky** [comment]
@fs-eire yes I think webgl is very hardware dependent. The webgl in Android emulator should be using either cpu-emulated webgl, or the Intel webgl backend.  So was trying to find out if there is a minimum comformant requirement when running mobile.

Would suggest Pixel devices as target device as it is being developed by Google.  It is just odd that Conv2d does not work on Huawei device.  Gonna try to find more Android devices for testing Conv2d.
